org.uqbar.project.wollok.ui.diagrams/src/org/uqbar/project/wollok/ui/diagrams/classes/StaticDiagramView.xtend

```diff
@@ -584,7 +584,7 @@ class StaticDiagramView extends ViewPart implements ISelectionListener, ISourceV
 	
 	/* Static Diagram Configuration Notifications */
 	override update(Observable o, Object event) {
-		if (event?.equals(StaticDiagramConfiguration.CONFIGURATION_CHANGED)) {
+		if (event !== null && event.equals(StaticDiagramConfiguration.CONFIGURATION_CHANGED)) {
```

Comment:
In the future we can use the `notNullAnd` combinator I created in a recent PR.

Reply:
:1st_place_medal: !!